### DISCUSSION TITLE
Ore-Control with Terra

### DISCUSSION THREAD

**Discussion by DraceRad** [DerFrZocker/Ore-Control#17] [Q&A]
Hello. I wanted to bother you with a question regarding your "Ore Control" plugin. I am working on a server with the generation of the world from another plugin that you may have heard of - Terra. However, it became a problem for me to implement the desired idea within Terra: the distribution of ore by biomes. Therefore, I turned to the help of your creation. My question: is it possible to generate a world using Terra and your plugin at the same time?
P.s. Game version 1.17.1

**Comment by DerFrZocker** [comment]
This depends on how Terra generates those ores. After a quick look at the code of Terra, Ore Control should be able to change it.

However Ore Control only supports custom biomes (which Terra makes use of, if I'm mistaken) since the rewrite, which limited the plugin to Minecraft versions 1.18 and higher.

Older versions of Ore Control which support 1.17 do not supported custom biomes.
